src/util/serverUtils.spec.ts

```diff
@@ -187,12 +187,16 @@ describe('groupServers', () => {
       [false, false],
     ];
 
-    const servers = props.map(([isManaged, isRunning], i) => ({
-      type: 'DHC' as const,
-      url: new URL(`http://localhost:1000${i}`),
-      isManaged,
-      isRunning,
-    }));
+    const servers = props.map(
+      ([isManaged, isRunning], i) =>
+        ({
+          type: 'DHC' as const,
+          url: new URL(`http://localhost:1000${i}`),
```

Comment:
Kind of limits to 10 servers, which you _probably_ won't hit (why would you bother making so many servers anyway?), but you could avoid that issue by doing something like:
```
`http://localhost:${10000 + i}`
```

Reply:
this is just the unit test. The actual range is explicitly set to 4 by default. I had imagined eventually exposing a config option to specify the range in case the defaults conflict with other ports.